java/src/jmri/jmrit/operations/rollingstock/RollingStock.java

```diff
@@ -813,7 +815,7 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
                         if (e.getNewValue() != null) {
                             Location newLocation = locationManager.getLocationByReporter((jmri.Reporter) e.getNewValue());
                             if (newLocation != getLocation())
-                                setLocation(newLocation, null);
+                                setWhereLastSeen(newLocation);
```

Comment:
On Aug 10, 2016, at 7:14 PM, Daniel Boudreau <notifications@github.com> wrote:
> Paul, Can you provide me with a copy of your script? I’d like to pass it along to the user
> who requested these changes.

Sure, but I will need to add a few comments to it so that it is understandable.

I'll work on that tonight, after my daughter is in bed.

Paul